src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+a {
+  text-decoration: none;
+}
\ No newline at end of file
```

Comment:
nit: Add a newline at the end of file to maintain consistent formatting across files. 📏